pkg/scanners/trivy/trivy.go

```diff
@@ -240,5 +247,48 @@ func main() {
 		os.Exit(generalErr)
 	}
 
+	// configure scanner metrics
+	ctxB := context.Background()
+	ctx, cancel := signal.NotifyContext(ctxB, os.Interrupt, syscall.SIGTERM)
+	defer cancel()
+
+	exporter, err := otlpmetrichttp.New(ctx, otlpmetrichttp.WithInsecure(), otlpmetrichttp.WithEndpoint(os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")))
```

Comment:
can this use ConfigureMetrics?

Reply:
scanner and eraser don't have a shared util with the imagelist and imagecollector controller, but I can add those to the pkg/util for them to share!